staging/src/k8s.io/apiserver/plugin/pkg/audit/log/backend.go

```diff
@@ -34,60 +35,69 @@ const (
 	FormatJson = "json"
 )
 
+// The plugin name reported in error metrics.
+const pluginName = "log"
+
 // AllowedFormats are the formats known by log backend.
 var AllowedFormats = []string{
 	FormatLegacy,
 	FormatJson,
 }
 
-type backend struct {
+func NewBackend(out io.Writer, format string, groupVersion schema.GroupVersion) (audit.Backend, error) {
+	return newBlockingBackend(out, format, groupVersion), nil
+}
+
+type blockingBackend struct {
```

Comment:
Why rename it and changing anything in this file except for the synchronization?

Reply:
I meant to differentiate with bufferedBackend